samples/crypto/aes_gcm/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -13,3 +13,7 @@ CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED=y
 
 # Disable Data Cache
 CONFIG_DCACHE=n
+
+# Mbedtls configuration
+CONFIG_MBEDTLS_ENABLE_HEAP=y
```

Comment:
For the 54h20 this should not be needed. At least if the SDFW is being used this should not be enabled I think. You cannot locally check it now because of some other issues but I would propose to check if we can remove it when the new version of the SDFW is released.